### DISCUSSION TITLE
Filter out "own ip"

### DISCUSSION THREAD

**Discussion by Caroga** [plausible/analytics#594] [Anything Else]
Hi there!
I've been browsing my site to check if everything is alright but I find that my visits are also being registered (ofcourse, duh!)
I was wondering if I could filter out visits that are not mine (ip based).
This could be done at the frontend part by determining the ip and decide rather or not to include the plausible lib but I was thinking this would logically fit into the dashboard too.
Does this make any sense?

**Update**: It’s now easy to exclude your internal traffic from being counted in your stats. See how here: https://plausible.io/docs/excluding

**Comment by metmarkosaric** [comment]
It’s now easy to exclude your internal traffic from being counted in your stats. See how here: https://plausible.io/docs/excluding

**Reply by Caroga** [reply]
Hi @metmarkosaric, thank you for your help.
I've tried adding the following rule to adguard but it's not helping. I might be confusing the rule so maybe you might be able to help me understand this.
I've added the following rule:
```
||blog.caroga.net^$script,domain=analytics.caroga.net
```
If I understand correctly I should be listing the domain where the script is injected into as first entry where I use the $script modifier to disable loading resources from the second domain, in this case the analytics subdomain.
Is this correct?

**Reply by metmarkosaric** [reply]
you should do it the other way around. try something like this please and let me know if it works:

```||analytics.caroga.net^$script,domain=blog.caroga.net```

**Reply by Caroga** [reply]
Hi! I've added this to my adguard home instance and it didn't work.
Adding this to the adblocker chrome plugin works. 
I will search and try to find a way to have it on adguard as well. For now this is fine.
Really looking forward to being able to register my home ip address and have the filteron on a network level instead of per device configuration.

Thank you for your help!

**Reply by metmarkosaric** [reply]
thanks for the update! not sure how the home instance works but glad you got it working with the browser extension. we'll keep you posted on the IP blocking feature!

**Comment by metmarkosaric** [comment]
**Update**: It’s now easy to exclude your internal traffic from being counted in your stats. See how here: https://plausible.io/docs/excluding
